### PR TITLE
added minidash. adjusted chart logic

### PR DESCRIPTION
**Pull request by staceyfras**
added mini dash for labor costs + gross sales to employees.html + inventory.html. adjusted views. reports.html is just my playground atm will not affect other files.

### CODE REVIEW

File: sporkify/frontend/templates/employees.html [hunk 1]
```diff
@@ -91,6 +92,38 @@
 		<input type="submit" id="clockInOut" name="clockInOut" value='Clock {{ clockedIn|yesno:"Out,In" }}' />
 	</form>
 </div>
+
+<div id="laborcostContainer">
+    <canvas id="doughnut-chart" width="500" height="300"></canvas>
+    <script>
+        var dv = document.getElementById("doughnut-chart")
+        var dchart = new Chart(dv, {
+            type: 'doughnut',
```
Comment (joshuaferrara) [inline sporkify/frontend/templates/employees.html]: I like doughnuts

File: sporkify/frontend/templates/inventory.html [hunk 2]
```diff
@@ -49,7 +50,7 @@
             </li>
             <li id="bullet4_purchased">
                 <label id="purVendor">Vendor<br>
-                    <select id="vendor" name="vendor">
+                    <select id="endor" name="vendor">
```
Comment (joshuaferrara) [inline sporkify/frontend/templates/inventory.html]: image
image

Endor has the best Ewoks

File: sporkify/frontend/views.py [hunk 3]
```diff
@@ -33,6 +45,8 @@ def employee(request):
     # Dictionary of stuff that should always be included in the
     # render context
     base_context = {
+        "labor_cost" : labor_costs(),
```
Comment (joshuaferrara) [inline sporkify/frontend/views.py]: nice

File: sporkify/frontend/templates/reports.html [hunk 4]
```diff
@@ -1,44 +1,50 @@
...
+		
 		{% block headIncludes %}
 		<script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.7.0/Chart.min.js"></script>
 		<title>Sporkify - Reports</title>
 		{% endblock %}
 		{% block content %}
-		<canvas id="myChart" width="50" height="200"></canvas>
+		</script>
+		<div id="BarChart">
+		<canvas id="myChart" width="500" height="200"></canvas>
```
Comment (joshuaferrara) [inline sporkify/frontend/templates/reports.html]: Irl I'd name these something relevant, but for this it's fine.

File: sporkify/frontend/templates/inventory.html [hunk 5]
```diff
@@ -76,6 +77,47 @@
 
     </form>
 
+    <div id="MiniHistoryGross">
+        <a href="/reports" target="_blank" style="color:black"><h1>Sales Reports</h1></a>
```
Comment (joshuaferrara) [inline sporkify/frontend/templates/inventory.html]: Did you mean for this sales chart to be on the inventory (unsold) page?

File: sporkify/frontend/templates/employees.html (hunk 1 above)
Comment (staceyfras) [reply sporkify/frontend/templates/employees.html]: I LOVE DONUTSS! yeah im probably going to update the naming conventions too.

File: sporkify/frontend/views.py (hunk 3 above)
Comment (staceyfras) [reply sporkify/frontend/views.py]: YA KNOW!

File: sporkify/frontend/templates/inventory.html (hunk 5 above)
Comment (staceyfras) [reply sporkify/frontend/templates/inventory.html]: ... i dunno.. TBD? i have to make 4 more charts sooo it'll find a home 🥉

File: sporkify/frontend/templates/inventory.html (hunk 2 above)
Comment (staceyfras) [reply sporkify/frontend/templates/inventory.html]: ... what a dork

Comment (staceyfras) [reply sporkify/frontend/templates/inventory.html]: in the BEST way possible!

Comment (joshuaferrara) [reply sporkify/frontend/templates/inventory.html]: :nerd_face:

**Pull request merged**